tests/AppInfo/ApplicationTest.php

```diff
@@ -80,44 +80,9 @@ public function testContainerAppName(): void {
 
 	public function queryData(): array {
 		return [
-			[IL10N::class],
 			[View::class],
-
-			// lib/
-			[Capabilities::class],
-			[Capabilities::class, ICapability::class],
-			[Consumer::class],
-			[Consumer::class, IConsumer::class],
-			[CurrentUser::class],
 			[Data::class],
-			[FilesHooks::class],
-			[GroupHelper::class],
-			[GroupHelperDisabled::class],
-			[GroupHelperDisabled::class, GroupHelper::class],
 			[MailQueueHandler::class],
-			[Navigation::class],
-			[UserSettings::class],
-			[ViewInfoCache::class],
-
-			// BackgroundJob
-			[EmailNotification::class],
-			[EmailNotification::class, TimedJob::class],
-			[ExpireActivities::class,],
-			[ExpireActivities::class, TimedJob::class],
-
-			// Controller
-			[ActivitiesController::class],
-			[ActivitiesController::class, Controller::class],
-			[APIv1Controller::class],
-			[APIv1Controller::class, Controller::class],
-			[APIv1Controller::class, OCSController::class],
-			[APIv2Controller::class],
-			[APIv2Controller::class, Controller::class],
-			[APIv2Controller::class, OCSController::class],
-			[FeedController::class],
-			[FeedController::class, Controller::class],
-			[SettingsController::class, SettingsController::class],
-			[SettingsController::class, Controller::class],
```

Comment:
How come we don't need that anymore?

Reply:
It uses normal DI, so should be fine.
The entries that I kept uses custom services defined in application class.
But we can also keep them. then let me fix the TimedJob one

Reply:
If this is testing the DI logic, then it's unrelated and I am fine with not keeping it :)